tests/Operation/AggregateFunctionalTest.php

```diff
@@ -217,7 +217,17 @@ function() use ($pipeline, $options) {
                 $results = iterator_to_array($operation->execute($this->getPrimaryServer()));
 
                 $this->assertCount(1, $results);
-                $this->assertObjectHasAttribute('stages', current($results));
+                $result = current($results);
+
+                if ($this->isShardedCluster()) {
```

Comment:
We had sharded clusters in Travis before this PR, so was this test previously failing?

Reply:
I'm not sure why it started failing, I'll try to narrow it down to a certain commit. The difference made sense so I thought I'd update the test, but it did seem strange. I'll see what I can find out.

Reply:
That's why I originally wanted to leave a comment on this yesterday, because I knew that I'd forget overnight. The test change is necessary as the output of `explain` is different in MongoDB 4.2. We might want to further adapt the test and check the server version as well, but for now this will do.